rm_vk/rm_vk/NewsFeedViewController/ViewController/NewsFeedViewController.swift

```diff
@@ -0,0 +1,128 @@
+// NewsFeedViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Гланый экран новостей
+final class NewsFeedViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let headerTableViewCellIdentifier = "HeaderTableViewCell"
+        static let postTextTableViewCellIdentifier = "PostTextTableViewCell"
+        static let postPhotoTableViewCellIdentifier = "PostPhotoTableViewCell"
+        static let footerTableViewCellIdentifier = "FooterTableViewCell"
+    }
+
+    // MARK: - IBOutlet
+
+    @IBOutlet private var tableView: UITableView!
+
+    // MARK: - Private Properties
+
+    private let networkService: NetworkServiceProtocol = NetworkService()
```

Comment:
```suggestion
    private let networkService = NetworkService()
```
а для чего здесь явное указание типа?